images/.gitignore

```diff
@@ -0,0 +1,10 @@
+*.pyc
```

Comment:
I think we can merge this one with the top-level `.gitignore`

Reply:
Done